app/views/employers/census_employees/new.html.erb

```diff
@@ -1,4 +1,4 @@
-<% content_for :navigation do %>
+<% content_for :left_navigation do %>
   <%= render :partial => 'employers/employer_profiles/primary_nav', :locals => {:active_tab=>params[:tab], :employer_profile_id=>params[:employer_profile_id]} %>
 <% end %>
 
```

Comment:
This form_for should be wrapped inside a content block just like the left navigation